src/python/grpcio/tox.ini

```diff
@@ -15,8 +15,5 @@ commands =
     coverage html --include='grpc/*' --omit='grpc/framework/alpha/*','grpc/early_adopter/*','grpc/framework/base/*','grpc/framework/face/*','grpc/_adapter/fore.py','grpc/_adapter/rear.py'
     coverage report --include='grpc/*' --omit='grpc/framework/alpha/*','grpc/early_adopter/*','grpc/framework/base/*','grpc/framework/face/*','grpc/_adapter/fore.py','grpc/_adapter/rear.py'
 deps =
-    cython
-    coverage
-    oauth2client
-    protobuf
+    -rrequirements.txt
```

Comment:
The absence of space between `-r` and `requirements.txt` on this line is a legitimate thing and not a typo, right?